code/game/objects/items/stacks/medical.dm

```diff
@@ -165,3 +166,91 @@
 /obj/item/stack/medical/ointment/suicide_act(mob/living/user)
 	user.visible_message("<span class='suicide'>[user] is squeezing \the [src] into [user.p_their()] mouth! [user.p_do(TRUE)]n't [user.p_they()] know that stuff is toxic?</span>")
 	return TOXLOSS
+
+/obj/item/stack/medical/splint
+	name = "medical splints"
+	singular_name = "medical splint"
+	icon = 'modular_skyrat/icons/obj/medical.dmi'
+	icon_state = "splint"
+	desc = "Oofie ouchie my bones."
+	novariants = TRUE
+	self_delay = 100
+	amount = 5
+	max_amount = 5
+
+/obj/item/stack/medical/splint/heal(mob/living/M, mob/user)
+	if(ishuman(M))
+		return heal_carbon(M, user)
+
+/obj/item/stack/medical/splint/heal_carbon(mob/living/carbon/C, mob/user, brute, burn)
+	if(ishuman(C))
+		var/mob/living/carbon/human/H = C
+		var/obj/item/bodypart/affecting = H.get_bodypart(user.zone_selected)
+
+		if(!(affecting in list(BODY_ZONE_R_ARM, BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_L_ARM, BODY_ZONE_PRECISE_L_HAND,\
+										BODY_ZONE_R_LEG, BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_L_LEG, BODY_ZONE_PRECISE_L_FOOT)))
+			to_chat(user, "<span class='danger'>You can't apply a splint there!</span>")
+			return FALSE
+		
+		if(affecting.status == BODYPART_ROBOTIC)
+			to_chat(user, "<span class='danger'>[H]'s [affecting] can't be broken, it's robotic!</span>")
+			return FALSE
+
+		if(affecting.status & BODYPART_SPLINTED)
+			to_chat(user, "<span class='danger'>[H]'s [affecting] is already splinted!</span>")
+			if(alert(user, "Would you like to remove the splint from [H]'s [affecting]?", "Splint removal.", "Yes", "No") == "Yes")
+				affecting.status &= ~BODYPART_SPLINTED
+				H.handle_splints()
+				to_chat(user, "<span class='notice'>You remove the splint from [H]'s [affecting].</span>")
+			return FALSE
+
+		user.visible_message("<span class='notice'>[user] applies [src] to [H]'s [affecting].</span>", \
+								"<span class='notice'>You apply [src] to [H]'s [affecting].</span>")
+
+		affecting.status_flags |= BODYPART_SPLINTED
+		affecting.splinted_count = world.time
+		H.handle_splints()
+		return TRUE
+	return FALSE
+
+/obj/item/stack/medical/splint/tribal
+	name = "tribal splints"
+	singular_name = "tribal splint"
+	icon_state = "tribal_splint"
+	desc = "Ooga booga rock crush bone."
+	self_delay = 200
+
+/obj/item/stack/medical/nanopaste
+	name = "nanite paste"
+	singular_name = "nanite paste"
+	icon = 'modular_skyrat/icons/obj/medical.dmi'
+	icon_state = "nanopaste"
+	desc = "01110111 01100101 01100101 01100100 00100000 01100101 01100001 01110100 01100101 01110010."
```

Comment:
"weed eater" isn't exactly a good description of something you use to repair artificial limbs and organs.

Reply:
I thought it was a mildly funny veggietales reference but yeah, i'm gonna change it.